homeassistant/components/plex/config_flow.py

```diff
@@ -60,6 +61,18 @@ def configured_servers(hass):
     }
 
 
+async def async_discover(hass):
+    """Scan for available Plex servers."""
+    gdm = GDM()
+    await hass.async_add_executor_job(gdm.scan)
+    for server_data in gdm.entries:
+        await hass.config_entries.flow.async_init(
+            DOMAIN,
+            context={"source": config_entries.SOURCE_DISCOVERY},
```

Comment:
We should use another source. This source is already reserved for the `discovery` integration.

Reply:
Should I create a new one? Any suggestions in mind? Maybe "discovery requested" or "onboarding"?

Reply:
integration_discovery maybe? Or self discovery